### PR TITLE
Adds the new pun pun to the maps

### PR DESCRIPTION
**Pull request by Incoming5643**
Was supposed to be with #15564 but it was merged 2fast4me.

This also adds a bunch of extra monkey names.

### CODE REVIEW

**Comment by RemieRichards** [comment]
3 is a better game tbh

**Comment by PKPenguin321** [comment]
I made a PR on your last branch with like 30 monkey names from the forums. Why don't you just use that?

**Comment by CosmicScientist** [comment]
Don't forget the monkey names on the closed PR too!

**Comment by Incoming5643** [comment]
I consolidated (most) of the ape names and added them.

File: code/modules/mob/living/carbon/monkey/punpun.dm [hunk 1]
```diff
@@ -6,7 +6,8 @@
 	var/relic_hat	//Note: these two are paths
 	var/relic_mask
 	var/memory_saved = 0
-	var/list/pet_monkey_names = list("Pun Pun", "Bubbles", "Mojo", "Forever")
+	var/list/pet_monkey_names = list("Pun Pun", "Bubbles", "Mojo", "George", "Darwin", "Aldo", "Caeser", "Kanzi", "Kong", "Terk", "Grodd", "Mala", "Bojangles", "Coco", "Able", "Baker", "Scatter")
```
Comment (KazeEspada) [inline code/modules/mob/living/carbon/monkey/punpun.dm]: I don't see Travis.

**Comment by CosmicScientist** [comment]
:+1:

**Comment by KazeEspada** [comment]
Maybe some lizard monkey names(since 50% of the time the bartender is a lizard anyways). Flings-the-poop. Rips-the-face. Bites-the-people.

**Comment by Incoming5643** [comment]
Not all servers have lizards, and not all servers that have lizards follow the Verbs-the-noun convention (which isn't even the convention the actual code suggests).

**Comment by KazeEspada** [comment]
But the names are still funny.

**Pull request merged**